pkg/security/podsecuritypolicy/factory.go

```diff
@@ -47,6 +47,11 @@ func (f *simpleStrategyFactory) CreateStrategies(psp *policy.PodSecurityPolicy,
 		errs = append(errs, err)
 	}
 
+	groupStrat, err := createGroupStrategy(psp.Spec.RunAsGroup)
```

Comment:
I expected this omitted if the feature gate was off (leave `groupStrat` as nil if the feature is disabled)